src/app/(site analysis)/sites/page.tsx

```diff
@@ -77,6 +78,12 @@ export default async function Page() {
         <Button asChild>
           <Link href="/sites/new">Register New Construction Site</Link>
         </Button>
+        <InfoBox>
+        <div>
+        If this is your first time login, click on the Resgister New Construction Site button above to start. <br />
+        Else click on the project list below to continue.
```

Comment:
we can change this to distinguish between supervisors and clients

e.g. tell clients to create a new site, and tell supervisors to wait until the clients add them to the site project